src/main/java/Duke.java

```diff
@@ -1,10 +1,51 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner scanner = new Scanner(System.in);
+        TaskList tasks = new TaskList();
+        tasks.loadTasksFromFile();
+        Runtime.getRuntime().addShutdownHook(new Thread(tasks::saveTasksToFile));
+
+        System.out.println("____________________________________________________________");
+        System.out.println(" Hello! I'm Axela");
+        System.out.println(" What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        while (true) {
+            String command = scanner.nextLine();
+            System.out.println("____________________________________________________________");
```

Comment:
Is it possible to abstract all the printing of lines into a variable that can be used throughout your code?